### PR TITLE
feat: use eth-network-package to spin up participant network

### PR DESCRIPTION
**Pull request by tedim52**

### CODE REVIEW

**Review by leoporoli — APPROVED** [review]
Approved with couple of comments to review.

File: main.star [hunk 1]
```diff
@@ -17,6 +18,9 @@ GRAFANA_DASHBOARD_PATH_URL = "/d/QdTOwy-nz/eth2-merge-kurtosis-module-dashboard?
 FIRST_NODE_FINALIZATION_FACT = "cl-boot-finalization-fact"
 HTTP_PORT_ID_FOR_FACT = "http"
 
+MEV_BOOST_SERVICE_NAME_PREFIX = "mev-boost-"
```
[the remark this answers is not in the document]
Comment (tedim52) [reply main.star:21]: I believe this is is just a prefix similar to these in `eth-network-package`
```
CL_CLIENT_SERVICE_NAME_PREFIX = "cl-client-"
EL_CLIENT_SERVICE_NAME_PREFIX = "el-client-"
```

File: main.star [hunk 2]
```diff
@@ -30,14 +34,23 @@ def run(plan, args):
...
 		all_cl_client_contexts.append(participant.cl_client_context)
-
+	
+	# spin up mev boost contexts
+	all_mevboost_contexts = []	
+	for index, participant in enumerate(args_with_right_defaults.participants):
+			mev_boost_context = None
+			if hasattr(participant, "builder_network_params") and participant.builder_network_params != None:
+				mev_boost_launcher = mev_boost_launcher_module.new_mev_boost_launcher(MEV_BOOST_SHOULD_CHECK_RELAY, participant.builder_network_params.relay_endpoints)
+				mev_boost_service_name = MEV_BOOST_SERVICE_NAME_PREFIX.format(1)
```
[the remark this answers is not in the document]
Comment (tedim52) [reply main.star:51]: good catch!

File: main.star (hunk 1 above)
[the remark this answers is not in the document]
Comment (leoporoli) [reply main.star:21]: but if you don't add the `{}` in the string how this statement will add the index? `MEV_BOOST_SERVICE_NAME_PREFIX.format(index)` https://qri.io/docs/reference/starlark-language/built-in-methods/string#format

**Pull request merged**